src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+@import './variables';
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Don't use * selector (it impacts performance) better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_calendar/assets/54860641/73b4c515-fc16-4335-86ec-02a3aabd1e7c)